services/nginz/integration-test/conf/nginz/nginx.conf

```diff
@@ -118,6 +118,7 @@ http {
 
     zauth_keystore resources/zauth/pubkeys.txt;
     zauth_acl conf/nginz/zauth_acl.txt;
+    oauth_key resources/oauth/ed25519_public.jwk;
```

Comment:
```suggestion
    # needs to be kept in sync with services/brig/test/resources/oauth/ed25519.jwk
    oauth_key resources/oauth/ed25519_public.jwk;
```

Reply:
it has conflicts now, I'll add it manually.